discovery-provider/plugins/notifications/src/email/notifications/components/Body.tsx

```diff
@@ -105,6 +105,14 @@ const snippetMap = {
     const entityName = notification.entity.name
     return `${entityName} is now trending thanks to you! Great work 🙌🏽`
   },
+  ['usdc_purchase_seller'](notification) {
+    const { entity, users, amount } = notification
+    return `Congrats, ${users[0].name} just bought your track ${entity.name} for ${amount} USDC!`
```

Comment:
Did this copy come from product/design? Feels slightly inconsistent that we list the price without the token suffix (`$1.00`) in the UI but then we would add the token suffix USDC in the notification.

Reply:
[We have a function in client](https://github.com/AudiusProject/audius-client/blob/gated-content-updates/packages/common/src/utils/wallet.ts#L153). The general idea is to use bigint math and make sure that the first two decimals don't get chopped off.

Just a note: We're changing the representation of price in Track metadata to be in cents, not wei. So if you are reading values off of a transaction, you would divide by `1000000`, but if reading a price off of a track, divide by `100` instead.